### ISSUE TITLE
TALKBACK not working

### ISSUE THREAD

**Issue by Chimchimgli** [homebridge-eufy-security/plugin#369]
### What happened?

I cannot get Talkback to work at all on any of my Apple Devices like my Watch, iPhone, old iPad.

My setup includes a HomeBase 3 (S380), and some EufyCam 3's (S330).

Is there any configuration I have to do to make it work? 
I know the Talkback feature on this Eufy Homebridge plugin works as I had a Dual 2K Doorbell that worked perfectly, however I have since had to return it as I got another doorbell.

APPRECIATE any help and assistance... this is the only function that is not working for me from the native eufy app. Streaming looks great and as well as automations on my RPi 4 8GB.

Also just want to say BIG thanks for creating this plugin as it has reenabled all these useful features that I sorely missed when I had native HomeKit support from my previous EufyCam 2 cameras!

### Device Type

Indoor Camera (Supported)

### Plugin Version

beta (Best Effort)

### HomeBridge Version

v1.6.x (Supported)

### NodeJS Version

v18 (Supported)

### Operating System

Raspbian 10+ (Supported)

### Relevant log output

```shell
[8/17/2023, 11:26:46 PM] [EufySecurity-2.2.2] INFO:  HB3 station livestream (P2P session) for Front has started.
[8/17/2023, 11:26:46 PM] [EufySecurity-2.2.2] INFO:  [Front] [Video Process] Starting video stream: native
[8/17/2023, 11:26:46 PM] [EufySecurity-2.2.2] INFO:  [Front] [Audio Process] Starting audio stream: 16 kHz, 24 kbps, codec: libfdk_aac
[8/17/2023, 11:26:46 PM] [EufySecurity-2.2.2] INFO:  [Front] [Talkback Process] Starting audio stream: 16 kHz, 20 kbps, codec: libfdk_aac
```

### config

```shell
"enableCamera": "true",
                    "videoConfig": {
                        "audio": true,
                        "debug": true,
                        "vcodec": "copy",
                        "useSeparateProcesses": true,
                        "audioSampleRate": 16
                    },
                    "talkback": true,
                    "enableButton": false,
                    "motionButton": false,
                    "unbridge": true,
                    "refreshSnapshotIntervalMinutes": 15
```

**Comment by Chimchimgli** [comment]
I know it's saying it's still experimental - same thing using the Homebridge camera.ui plugin with some of my other non-Eufy cameras, Talkback feature maybe more complicated to include?

**Comment by lenoxys** [comment]
just tested on my setup and it's work I've a battery door bell. can you update on 2.2.5 and enable debug and sent me again the output ? thanks !

https://github.com/homebridge-eufy-security/plugin/wiki/Basic-Troubleshooting

**Comment by Chimchimgli** [comment]
Hi, didn't realize I got a reply from one of the actual contributors of this plug-in, my bad!

So I actually still have my Dual 2K Battery Powered doorbell, added it back temporarily to my Eufy guest/admin account and finally on Homebridge, where it appeared automatically (I actually ran into some issues and did a complete reset of the plugin and am currently on the beta v.2.3.0). And yes talkback on the doorbell does work indeed, but I cannot for some reason get it to work on any of my 4 EufyCam 3's (S330). 
I tried the non beta version but everything would become inaccessible, as in I couldn't change any settings as all my cams were 'locked'. I forget what version it was, but I see now there seems to be a lot of newer versions, though I'm hesitant to upgrade to them when everything - except Talkback, ofc, has been working great on v.2.3.0. Like if it ain't broke, dont fix it kind of thing... though I am curious if anyone with the S330's has gotten Talkback to work on them.

And I apologize I didn't even save the debug log, though I can reproduce it if needed. 
But a quick description of what the log outputted, it would say something like HKSV process began, video first, then audio, and Talkback process with the codec and sample rate. Ugh maybe it would be better if I reproduced it then come back here and paste it here.

But yea I really appreciate your response, and I should be back here soon with the debug log.

**Comment by Chimchimgli** [comment]
[9/6/2023, 5:39:02 PM] [EufySecurity-2.3.0] INFO:  HB3 station livestream (P2P session) for Balo has started.
[9/6/2023, 5:39:02 PM] [EufySecurity-2.3.0] INFO:  [-] [Video Process] Starting video stream: native
[9/6/2023, 5:39:02 PM] [EufySecurity-2.3.0] INFO:  [-] [Audio Process] Starting audio stream: 16 kHz, 24 kbps, codec: libfdk_aac
[9/6/2023, 5:39:02 PM] [EufySecurity-2.3.0] INFO:  [- [Talkback Process] Starting audio stream: 16 kHz, 20 kbps, codec: libfdk_aac
[9/6/2023, 5:39:34 PM] [EufySecurity-2.3.0] INFO:  - Stopped video stream.
[9/6/2023, 5:39:34 PM] [EufySecurity-2.3.0] INFO:  HB3 station livestream for Balo has stopped.
[9/6/2023, 5:39:36 PM] [EufySecurity-2.3.0] INFO:  [-] [Video Process] FFmpeg exited with code: null and signal: SIGKILL (Forced)
[9/6/2023, 5:39:36 PM] [EufySecurity-2.3.0] INFO:  [-] [Audio Process] FFmpeg exited with code: null and signal: SIGKILL (Forced)
[9/6/2023, 5:39:36 PM] [EufySecurity-2.3.0] INFO:  [-] [Talkback Process] FFmpeg exited with code: null and signal: SIGKILL (Forced)

So that's usually what it outputs, and my setup is:
RPi 4 8GB
1 HomeBase 3 with four EufyCam 3's; only one is enabled for HKSV, on the native Eufy app my settings for this one camera is Streaming Quality: Medium, Recording Quality: Full HD (1080P). I have an Apple TV 4K (2022) connected to Ethernet as my Home hub if that matters.

Oh and here's my config on Homebridge:
          {
                    "serialNumber": "T81**********",
                    "enableCamera": "true",
                    "videoConfig": {
                        "audio": true,
                        "debug": true,
                        "vcodec": "copy",
                        "useSeparateProcesses": true
                    },
                    "enableButton": false,
                    "motionButton": false,
                    "unbridge": true,
                    "refreshSnapshotIntervalMinutes": 15,
                    "hsv": true,
                    "hsvRecordingDuration": 150,
                    "hsvConfig": {
                        "crop": false
                    },
                    "talkback": true,
                    "talkbackChannels": 1
                },

If there's any more info needed, I would be more than happy to provide. One more thing, I tried the Stereo option in the plugin as well, but nada. Anyway, thanks!

**Comment by lenoxys** [comment]
Ok sorry this is the beta version so. Please I'm focusing on HKSV for the master which will be a different matter. I do have talkback on 2.2.12 but there is no HKSV on it. Can you let me know if you can test this version or prefer to wait ?

**Comment by Chimchimgli** [comment]
You know I’m just glad that you good people are continuing the good work on this plugin, because I can honestly say if I hadn’t found out about HB and this eufy plugin, I would’ve given up/sold all my eufy devices. I think eufy owes you and all the other contributors a lot as I’m pretty sure they would’ve lost a lot of customers - myself included, if this plugin didn’t exist.

With that being said, I’m fine waiting for the master release, like I said I’m glad there’s actually one in the works. I did try various new releases, however no Talkback, but that’s totally fine. Streaming works great even from running HB on my little RPi4, and it works fine most of the time (good internet connection is key) throughout all my Apple devices. So please, no rush, I will be really looking forward for the master release, and thanks again for all the work you’ve done for us all.

**Comment by macos1234** [comment]
Talk back isn’t working for me either. I have no Audio at all in my cameras via HomeKit

**Comment by darkaeden** [comment]
No working here too (Pan Tilt 2k and Indoor Cam)

**Comment by lenoxys** [comment]
**Thank you for taking the time to report this issue.**

As part of a repository cleanup, we are closing issues that have been open for more than 12 months without resolution. A **beta version** of the plugin has been released that includes significant changes and improvements.

We encourage you to try the latest beta and see if your issue has been addressed. If the problem persists, please feel free to open a new issue with updated details and logs from the latest version.

Thank you for your patience and support!
